exercise-1/style.css

```diff
@@ -0,0 +1,149 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+}
+
+html,
+body {
+  height: 100vh;
+  margin: 0;
+  font-family: 'Lato', 'san-serif';
+}
+
+button {
+  display: none;
+}
+
+
+nav {
+  min-width: 150px;
+  display: flex;
+  justify-content: end;
+  height: 100vh;
```

Comment:
Be careful with using `100vh`, on some (actually most) mobile browsers, it includes the browser chrome in calculating it, so it won't actually be 100% of the viewport. Use `100%` instead.